test/test_cursor.py

```diff
@@ -1392,6 +1392,14 @@ def test_to_list_empty(self):
         docs = c.to_list()
         self.assertEqual([], docs)
 
+    def test_to_list_length(self):
+        coll = self.db.test
+        coll.insert_many([{} for _ in range(5)])
+        self.addCleanup(coll.drop)
+        c = coll.find()
+        docs = c.to_list(3)
+        self.assertEqual(len(docs), 3)
```

Comment:
Could you add another test with a small batchSize? like:
```python
        c = coll.find(batch_size=2)
        docs = c.to_list(3)
        self.assertEqual(len(docs), 3)
        docs = c.to_list(3)
        self.assertEqual(len(docs), 2)
```

Reply:
Done